CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+## [UNRELEASED]
+### Added
+- Added a changelog
+
+### Changed
+- create orders without a contract_id
+- add get operation for orders
```

Comment:
So to simplify and overcome this sorting problem I opt for getting rid of these categories completely and just list the changes.